static/to_compile/controllers/admin/suggestion_groupe_row_controller.ts

```diff
@@ -0,0 +1,185 @@
+import { Controller } from "@hotwired/stimulus"
+import * as Turbo from "@hotwired/turbo"
+
+export default class extends Controller<HTMLElement> {
+  static targets = ["fieldsValues", "fieldsGroups"]
+
+  declare readonly fieldsValuesTarget: HTMLInputElement
+  declare readonly fieldsGroupsTarget: HTMLInputElement
+
+  fieldDisplayedFocus(event: Event) {
+    const value = this.#getFieldsValues()
+    const field = (event.target as HTMLElement).dataset.field
+    if (!field || !(field in value)) {
+      console.error("Champs manquants")
+      return
+    }
+    const target = event.target as HTMLElement
+    target.textContent =
+      value[field]["updated_displayed_value"] || value[field]["displayed_value"]
+  }
+
+  fieldDisplayedBlur(event: Event) {
+    const fieldsValues = this.#getFieldsValues()
+    const field = this.#getField(event, fieldsValues)
+    if (field != null) {
+      fieldsValues[field]["updated_displayed_value"] = (
+        event.target as HTMLElement
+      ).textContent
+      this.#postFieldsValues(fieldsValues)
+    }
+  }
```

Comment:
Suggestion : refléter ce que fait la fonction. `blur` est l'événement auquel on réagit, mais pas ce qu'on fait des champs. 


```suggestion
  postFieldValue(event: Event) {
    const fieldsValues = this.#getFieldsValues()
    const field = this.#getField(event, fieldsValues)
    if (field != null) {
      fieldsValues[field]["updated_displayed_value"] = (
        event.target as HTMLElement
      ).textContent
      this.#postFieldsValues(fieldsValues)
    }
  }
```